src/web/crate_details.rs

```diff
@@ -496,7 +496,7 @@ pub(crate) async fn get_all_platforms(
     Extension(pool): Extension<Pool>,
     uri: Uri,
 ) -> AxumResult<AxumResponse> {
-    let is_crate_root = uri.path().starts_with("/crate/");
+    let is_crate_root = uri.path().contains("/crate/");
```

Comment:
I think there could potentially be subpaths that contain a `/crate/` directory, like the rendered source if using something like `#[path = "/crate/foo.rs"]`.

Reply:
Good point. I can put the whole `/-/menu/platforms/crate/` with `starts_with`. What do you think?

Reply:
That sounds good

Reply:
Done!